src/Swarm/TUI/View.hs

```diff
@@ -334,6 +308,37 @@ drawGameUI s =
   fr = s ^. uiState . uiFocusRing
   moreTop = s ^. uiState . uiMoreInfoTop
   moreBot = s ^. uiState . uiMoreInfoBot
+  showREPL = s ^. uiState . uiShowREPL
+  rightPanel = if showREPL then worldPanel ++ replPanel else worldPanel
```

Comment:
This might be a bit nicer (assuming we make `replPanel` just a Widget and not a list):
```suggestion
  rightPanel = worldPanel ++ [ replPanel | showREPL ]
```

Reply:
I think completely hiding the REPL panel is reasonable.  We might also like to try replacing it with something like `hBorder` or `hBorderWithLabel`?  Not sure which would look better.

Reply:
Hi @byorgey .  Here is the minimized REPL with a hBorder.

![image](https://user-images.githubusercontent.com/91670077/231035905-5827110c-5f4d-4456-b302-39a371cea8f9.png)

Reply:
@ussgarci I think that looks nice, though we probably shouldn't hard-code the color as cyan, in case we *e.g.* later allow the focus color to be customized.  Ideally, it will be either cyan or white (or whatever the focused/unfocused colors are) depending on whether the REPL is focused.  I like that you can still type in the REPL even when it is minimized so it makes sense to show whether it has focus.  Off the top of my head I don't remember how to do this but maybe you can figure out how we decide to draw the other windows depending on whether they are focused.

Reply:
Thanks for the feedback.  I have removed the hard-coded cyan and am now indicating the focus properly.